GitUI/NBugReports/BugReportInvoker.cs

```diff
@@ -68,8 +68,30 @@ void AppendIfNotEmpty(string? value, string designation)
             }
         }
 
+        private static void LogError(Exception exception, bool isTerminating)
+        {
+            string tempFolder = Path.GetTempPath();
+            string tempFileName = $"{AppSettings.ApplicationId}.{AppSettings.AppVersion}.{DateTime.Now.ToString("yyyyMMdd.HHmmssfff")}.log";
+            string tempFile = Path.Combine(tempFolder, tempFileName);
+
+            try
+            {
```

Comment:
I am getting paranoid:
```suggestion
            try
            {
                string tempFolder = Path.GetTempPath();
                string tempFileName = $"{AppSettings.ApplicationId}.{AppSettings.AppVersion}.{DateTime.Now.ToString("yyyyMMdd.HHmmssfff")}.log";
                string tempFile = Path.Combine(tempFolder, tempFileName);

```

Reply:
We can't do that - `tempFile` is used to form the exception message.